A/at_spi2_atk/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "at_spi2_atk"
+version = v"2.34.1"
+
+# Collection of sources required to build at-spi2-atk
+sources = [
+    "http://ftp.gnome.org/pub/gnome/sources/at-spi2-atk/$(version.major).$(version.minor)/at-spi2-atk-$(version).tar.xz" =>
+    "776df930748fde71c128be6c366a987b98b6ee66d508ed9c8db2355bf4b9cc16",
+    "./bundled"
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/at-spi2-atk-*/
+
+# Tests are failing to be built, just disable them
+atomic_patch -p1 ../patches/meson_build.patch
```

Comment:
When I'll have some time I'd like to understand why the tests are failing, but now I don't have that time and I just disabled tests.

Reply:
For the record, it turned out that one of the tests is using the `timespec` struct, which however is a POSIX feature: https://stackoverflow.com/a/3875233/2442087.  By default, the library is built with `c99` standard and the additional features are not enabled.

Reply:
So do you want to force `-std=gnu99` into the CFLAGS, or just continue to build with tests disabled?  Seems to me like this might be masking out important functionality without the `-std=gnu99` if the tests are failing.  If you want to just push this through as-is, that's fine with me; I trust your judgement.

Reply:
I'd naively expect compile-time failures if other extensions are used elsewhere, thus I'd suggest to just continue disabling the tests, that we don't care anyway since we're cross-compiling.